tools/setup_env.sh

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+# This script sets up the environment for the Ziggurat test suite.
+
+# Rippled files
+if [ -z $RIPPLED_BIN_PATH ]; then
+    echo "Aborting. Export RIPPLED_BIN_PATH before running this script."
+    exit 1
+fi
+RIPPLED_BIN_NAME="rippled"
+
+# Ziggurat config files
+ZIGGURAT_RIPPLED_DIR="$HOME/.ziggurat/ripple"
+ZIGGURAT_RIPPLED_SETUP_DIR="$ZIGGURAT_RIPPLED_DIR/setup"
+ZIGGURAT_RIPPLED_SETUP_CFG_FILE="$ZIGGURAT_RIPPLED_SETUP_DIR/config.toml"
+ZIGGURAT_RIPPLED_TESTNET_DIR="$ZIGGURAT_RIPPLED_DIR/testnet"
+ZIGGURAT_RIPPLED_STATEFUL_DIR="$ZIGGURAT_RIPPLED_DIR/stateful"
+
+setup_config_file() {
+    echo "--- Setting up configuration file"
+    echo "Creating $ZIGGURAT_RIPPLED_SETUP_CFG_FILE with contents:"
+    mkdir -p $ZIGGURAT_RIPPLED_SETUP_DIR
+    echo
+    echo "# Rippled installation path" > $ZIGGURAT_RIPPLED_SETUP_CFG_FILE
+    echo "path = \"$RIPPLED_BIN_PATH\"" >> $ZIGGURAT_RIPPLED_SETUP_CFG_FILE
+    echo "# Start command with possible arguments" >> $ZIGGURAT_RIPPLED_SETUP_CFG_FILE
+    echo "start_command = \"./$RIPPLED_BIN_NAME --silent\"" >> $ZIGGURAT_RIPPLED_SETUP_CFG_FILE
+
+    # Print file contents so the user can check whether the path is correct
+    cat $ZIGGURAT_RIPPLED_SETUP_CFG_FILE
+    echo
+}
+
+setup_ip_addresses() {
+    echo "--- Creating a package of IP addresses required for performance tests"
+    sudo python3 ./tools/ips.py --subnet 1.1.1.0/24 --file src/tools/ips.rs --dev_prefix test_zeth
+    if [ $(uname) == "Linux" ]; then
+        sudo python3 ./tools/ips.py --subnet 1.1.1.0/24 --file src/tools/ips.rs --dev lo
```

Comment:
Does it mean that on Linux it `ips.py` will be executed twice? First time on line 35 and second time here resulting in having the same subnet added to test_zeth devices and to lo device?

Reply:
Ah yes that's true, good catch! :+1:

Reply:
So the same would be on Mac - first time on 35 and second time on line 39.